website/content/en/preview/upgrade/upgrade-guide.md

```diff
@@ -1,127 +1,156 @@
 
-## Compatibility issues
+Some things that will help you with this upgrade include:
 
-To make upgrading easier, we aim to minimize the introduction of breaking changes with the following components:
+* **[Karpenter Upgrade Reference]({{< relref "upgrade-ref.md" >}})**: Provides a complete reference to help you transition your Provisioner, Machine, and AWSNodeTemplate manifests, as well as other components, to be able to work with the new v1beta1 names, labels, and other elements.
+* **[Karpenter conversion tool](https://github.com/aws/karpenter/tree/main/tools/karpenter-convert)**: Simplifies the creation of NodePool and EC2NodeClass manifests.
+* **[Compatibility]({{< relref "compatibility.md" >}})**: Offers details about Karpenter compatibility with different versions of Kubernetes and other components.
 
-* Provisioner API
-* Helm Chart
+##### Procedure
 
-We try to maintain compatibility with:
+This procedure assumes you are running the Karpenter controller on cluster and want to upgrade that cluster to v0.32.x.
 
-* The application itself
-* The documentation of the application
+**NOTE**: Please read through the entire procedure before beginning the upgrade. There are major changes in this upgrade, so you should carefully evaluate your cluster and workloads before proceeding.
 
-When we introduce a breaking change, we do so only as described in this document.
+1. Determine the current cluster version: Run the following to make sure that your Karpenter version is v0.31.x:
+   ```
+   kubectl get pod -A | grep karpenter
+   kubectl describe pod -n karpenter-xxxxxxxxxx-xxxxx | grep Image: | grep v0.....
+   ```
+   Sample output:
+   ```
+   Image: public.ecr.aws/karpenter/controller:v0.31.0@sha256:d29767fa9c5c0511a3812397c932f5735234f03a7a875575422b712d15e54a77
+   ```
 
-Karpenter follows [Semantic Versioning 2.0.0](https://semver.org/) in its stable release versions, while in
-major version zero (v0.y.z) [anything may change at any time](https://semver.org/#spec-item-4).
-However, to further protect users during this phase we will only introduce breaking changes in minor releases (releases that increment y in x.y.z).
-Note this does not mean every minor upgrade has a breaking change as we will also increment the
-minor version when we release a new feature.
+2. Review for breaking changes: If you are already running Karpenter v0.31, you can skip this step. If you are running an earlier Karpenter version, you need to review the upgrade notes each minor release to make sure that you deal with any breaking changes that might have been introduced. See [Release Upgrade Notes]({{< relref "#release-upgrade-notes" >}}) for details on each previous Karpenter version.
```

Comment:
Done.
I see that you renumbered this section. Were you aware that using "1." for each step will cause the whole procedure to be numbered sequentially? It will keep you from having to renumber a whole procedure if just one number changes.

Reply:
Done